beacon-chain/core/blocks/withdrawals.go

```diff
@@ -147,35 +146,17 @@ func ValidateBLSToExecutionChange(st state.ReadOnlyBeaconState, signed *ethpb.Si
 //	        next_index = state.next_withdrawal_validator_index + MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP
 //	        next_validator_index = ValidatorIndex(next_index % len(state.validators))
 //	        state.next_withdrawal_validator_index = next_validator_index
-func ProcessWithdrawals(st state.BeaconState, executionData interfaces.ExecutionData) (state.BeaconState, error) {
+func ProcessWithdrawals(st state.BeaconState) (state.BeaconState, error) {
+	if !st.IsParentBlockFull(){
```

Comment:
What happens if you call `IsParentBlockFull` on a state that's not epbs based, this will error out which is what we don't want

Reply:
yeah we need to handle error then here thanks

Reply:
should we check before this if condition 
if st.Version() >= version.EPBS {
		return nil, fmt.Errorf("Wrong version for withdrawals processing: %d", st.Version())
	}
I think that would work ?